### DISCUSSION TITLE
Imported SVG - how to change color?

### DISCUSSION THREAD

**Discussion by OnkelTem** [jgraph/drawio-desktop#1543] [General]
I don't see any way to change the color of an imported SVG. I import it just copy-pasting an SVG file in the editor.
However don't see the line style configuration available in the UI . So I can only change the background color. but not the line color which is rendered black-on-black.

**Comment by davidjgraph** [comment]
No, the SVG is just imported as an image. draw.io is not an SVG editing tool.
